torch/csrc/stable/tensor.h

```diff
@@ -118,6 +124,12 @@ class Tensor {
     return size;
   }
 
+  bool defined() const {
```

Comment:
This is the same semantic as the one in TensorBase.h right?

And to make sure I'm understanding correctly: an undefined tensor is an uninitialized tensor which I know has no memory but is it basically a nullptr?

Reply:
Yes and yes

My understanding is `TensorBase() = default` constructor would initialize its `c10::intrusive_ptr<TensorImpl, UndefinedTensorImpl> impl_` to a nullptr indeed